src/Stratis.Features.FederatedPeg/TargetChain/CrossChainTransferStore.cs

```diff
@@ -242,8 +239,8 @@ private ICrossChainTransfer[] ValidateCrossChainTransfers(ICrossChainTransfer[]
                     {
                         partialTransfer.SetPartialTransaction(walletTran);
 
-                        if (walletData[0].Item2.BlockHeight != null)
-                            tracker.SetTransferStatus(partialTransfer, CrossChainTransferStatus.SeenInBlock, walletData[0].Item2.BlockHash, (int)walletData[0].Item2.BlockHeight);
+                        if (walletData[0].Item2.BlockNumber != 0)
```

Comment:
Explain this change

Reply:
The `BlockNumber` on the `Withdrawal` object does not support null values. When `BlockHash` is null his value will be 0.